graph/db/kv_store.go

```diff
@@ -848,18 +848,19 @@ func (c *KVStore) SetSourceNode(node *models.LightningNode) error {
 //
 // TODO(roasbeef): also need sig of announcement.
 func (c *KVStore) AddLightningNode(node *models.LightningNode,
-	op ...batch.SchedulerOption) error {
+	options ...batch.SchedulerOption) error {
+
+	opts := batch.NewDefaultSchedulerOpts()
```

Comment:
nit: alternatively, we could add a `batch.NewSchedulerOptions(options ...batch.SchedulerOption)` constructor that applies the functional options during creation, so we don't have to duplicate them at the call sites.

Reply:
great idea - will add